scripts/environment.sh

```diff
@@ -57,6 +57,10 @@ export CAPM3_VERSION=${CAPM3_VERSION:-"v1beta1"}
 export M3PATH=${M3PATH:-"${HOME}/go/src/github.com/metal3-io"}
 export CAPM3_LOCAL_IMAGE="${CAPM3PATH}"
 
+export CAPM3_RELEASE = "v1.1.1"
```

Comment:
You can use `CAPM3RELEASE` directly since it is already exported from the sourced **devenv path** _`lib/releases.sh`_